rocketmq-spring-boot/src/test/java/org/apache/rocketmq/spring/support/DefaultRocketMQListenerContainerTest.java

```diff
@@ -46,6 +55,28 @@ public void onMessage(MessageExt message) {
         result = (Class) getMessageType.invoke(listenerContainer);
         assertThat(result.getName().equals(MessageExt.class.getName()));
     }
+
+    @Test
+    public void testGenericMessageType() throws Exception {
+        DefaultRocketMQListenerContainer listenerContainer = new DefaultRocketMQListenerContainer();
+        listenerContainer.setMessageConverter(new CompositeMessageConverter(Arrays.asList(new StringMessageConverter(), new MappingJackson2MessageConverter())));
+
+        Method getMessageType = DefaultRocketMQListenerContainer.class.getDeclaredMethod("getMessageType");
+        Method getMethodParameter = DefaultRocketMQListenerContainer.class.getDeclaredMethod("getMethodParameter");
+        getMessageType.setAccessible(true);
+        getMethodParameter.setAccessible(true);
+        listenerContainer.setRocketMQListener(new RocketMQListener<ArrayList<Date>>() {
+            @Override
+            public void onMessage(ArrayList<Date> message) {
+
+            }
+        });
+
+        ParameterizedType type = (ParameterizedType) getMessageType.invoke(listenerContainer);
+        assertThat(type.getRawType() == ArrayList.class);
+        MethodParameter methodParameter = ((MethodParameter) getMethodParameter.invoke(listenerContainer));
+        System.out.println(methodParameter);
```

Comment:
It would be better that do not call System.out.println in test method.

Reply:
already fixed it